projects/angular/src/data/stack-view/_stack-view.clarity.scss

```diff
@@ -34,45 +34,88 @@
     .stack-title {
       display: block;
       flex: 1 1 auto;
-      padding: variables.$clr_baselineRem_0_25 0;
+
+      @include mixins.css-var(
+        color,
+        clr-stack-view-title-color,
+        variables.$clr-h4-color,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        padding,
+        clr-stack-view-stack-title-padding,
+        variables.$clr_baselineRem_0_25 0,
+        variables.$clr-use-custom-properties
+      );
+
+      @include mixins.css-var(
+        font-weight,
+        clr-stack-view-stack-title-font-weight,
+        200,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        font-size,
+        clr-stack-view-stack-title-font-size,
+        0.9rem,
```

Comment:
This one would probably be better with a baselineRem variable, but as I imagine how you obtained this value (and it will soon go away) I won't frown too much if it stays.

Reply:
corrected in commit d7cd4ac